### PR TITLE
Added option to download decoded "signed" attachment

### PR DESCRIPTION
**Pull request by Yossi-a**

### CODE REVIEW

File: open-budget-il/main.py [hunk 1]
```diff
@@ -934,6 +977,8 @@ def get(self,code,year):
     ('/api/exemption/(new)', ExemptionsApi),
     ('/api/exemption/(updated)/([0-9][0-9][0-9][0-9])-([0-9][0-9])-([0-9][0-9])/([0-9][0-9][0-9][0-9])-([0-9][0-9])-([0-9][0-9])', ExemptionsApi),
 
+    ('/api/attachment', AttachmentsApi),
```
Comment (akariv) [inline open-budget-il/main.py]: Let's give it a more meaningful name (exemption signed document related).

File: open-budget-il/main.py [hunk 2]
```diff
@@ -431,6 +434,46 @@ def get_query(self,*args,**kw):
...
+
+        # Get rid of the goddamn namespaces.
+        namespace_re = re.compile(r'\{.*\}')
+        for elem in doc.iter():
+            elem.tag = namespace_re.sub('', elem.tag)
+
+        # Get the data and decode it.
+        data_elem = doc.find('./SignedObject/SignedInfo/Data')
+        data_b64 = data_elem.text
+        data = b64decode(data_b64)
```
Comment (akariv) [inline open-budget-il/main.py]: data_b64.decode('base64')

File: open-budget-il/main.py [hunk 3]
```diff
@@ -24,6 +25,8 @@
 from models import MRExemptionRecord, MRExemptionRecordDocument, MRExemptionRecordHistory
 from secret import ALLOWED_EMAILS, UPLOAD_KEY
 from upload import upload_handlers
+from base64 import  b64decode
```
Comment (akariv) [inline open-budget-il/main.py]: no need for this

**Comment by Yossi-a** [comment]
1. I've fixed the class name and removed the redundant dependency
2. It's also supports downloading regular files (not signed)

**Pull request merged**